packages/winpanda/extra/src/winpanda/svcm/exceptions.py

```diff
@@ -43,3 +43,13 @@ class ServiceConfigError(ServiceError):
 class ServiceSetupError(ServiceError):
     """Service setup error."""
     pass
+
+
+class ServiceFluctuantError(ServiceError):
```

Comment:
I had to look up "fluctuant" - it's not a common word. How about `ServiceTransientError` and `ServicePermanentError`?

Reply:
I was considering several words for this position: intermittent, temporary, unsteady, unstable, soft and ...  transient. OK 'transient' is probably better. But for the opposite I'd rather chose 'persistent' instead of 'permanent'. I like it better. 

Thus, we'd have **'ServiceTransientError'** and **'ServicePersistentError'** instead of the current pair - **'ServiceFluctuantError'** and **'ServiceHardError'**